plugins/nextcloud/index.php

```diff
@@ -10,6 +10,26 @@ class NextcloudPlugin extends \RainLoop\Plugins\AbstractPlugin
 		DESCRIPTION = 'Integrate with Nextcloud v20+',
 		REQUIRED = '2.36.2';
 
+	private const IGNORE_SYSTEM_ADDRESSBOOK_KEY = 'ignoreSystemAddressbook';
+	private const IGNORE_SYSTEM_ADDRESSBOOK_DEFAULT_VALUE = true;
+
+	private const ENABLE_NC_ADDRESSBOOK_KEY = 'enableNcAddressbook';
+	private const ENABLE_NC_ADDRESSBOOK_DEFAULT_VALUE = false;
+
+	private const DISABLE_INHOUSE_ADDRESSBOOK_KEY = 'disableSnappymailContactsUI';
+	private const DISABLE_INHOUSE_ADDRESSBOOK_DEFAULT_VALUE = false;
+
+	private const DEFAULT_ADDRESSBOOK_URI_KEY = 'defaultNCAddressbookUri';
+	private const DEFAULT_ADDRESSBOOK_URI = 'webmail';
+
+	private const DEFAULT_ADDRESSBOOK_NAME_KEY = 'defaultNCAddressbookName';
+	private const DEFAULT_ADDRESSBOOK_NAME = 'WebMail';
+
+	private const DEFAULT_ADDRESSBOOK_DESCRIPTION_KEY = 'defaultNCAddressbookDescription';
+	private const DEFAULT_ADDRESSBOOK_DESCRIPTION = 'Recipients from snappymail';
```

Comment:
@fahim44 Can you make this a translation key? :)

Reply:
@akhil1508 this is actually already done https://github.com/Murena-SAS/snappymail/pull/2/files#diff-bd3b167b234e3841b4b8426078057e931e0be4a9b948d1f0a825755a0022e231R506